src/app/pages/account/password/reset/index.vue

```diff
@@ -69,6 +89,19 @@ const { t } = useI18n()
 const title = t('title')
 useHeadDefault({ title })
 
+// stepper
+const { error, restart, step } = useStepperPage<
+  'default' | 'success' | 'error'
+>({
+  steps: {
+    default: {},
+    success: {},
+    error: {
+      title: t('globalError'),
+    },
+  },
+})
```

Comment:
If just the defaults are set, then simply:

```suggestion
const { error, restart, step } = useStepper<
  'default' | 'success' | 'error'
>()
```

Reply:
done, you can resolve once checked